parm/archive/master_enkf.yaml.j2

```diff
@@ -13,101 +10,26 @@ datasets:
-        {% set COMIN_OCEAN_ANALYSIS_MEM = COM_OCEAN_ANALYSIS_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_OCEAN_LETKF_MEM = COM_OCEAN_LETKF_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_OCEAN_HISTORY_MEM = COM_OCEAN_HISTORY_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_OCEAN_RESTART_MEM = COM_OCEAN_RESTART_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_ICE_ANALYSIS_MEM = COM_ICE_ANALYSIS_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_ICE_LETKF_MEM = COM_ICE_LETKF_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_ICE_HISTORY_MEM = COM_ICE_HISTORY_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_ICE_RESTART_MEM = COM_ICE_RESTART_TMPL | replace_tmpl(tmpl_dict) %}
-        {% set COMIN_MED_RESTART_MEM = COM_MED_RESTART_TMPL | replace_tmpl(tmpl_dict) %}
-
-        # Append the member COM directories
-        {% do COMIN_ATMOS_ANALYSIS_MEM_list.append(COMIN_ATMOS_ANALYSIS_MEM)%}
-        {% do COMIN_ATMOS_HISTORY_MEM_list.append(COMIN_ATMOS_HISTORY_MEM)%}
-        {% do COMIN_ATMOS_RESTART_MEM_list.append(COMIN_ATMOS_RESTART_MEM)%}
-        {% do COMIN_OCEAN_ANALYSIS_MEM_list.append(COMIN_OCEAN_ANALYSIS_MEM)%}
-        {% do COMIN_OCEAN_LETKF_MEM_list.append(COMIN_OCEAN_LETKF_MEM)%}
-        {% do COMIN_OCEAN_HISTORY_MEM_list.append(COMIN_OCEAN_HISTORY_MEM)%}
-        {% do COMIN_OCEAN_RESTART_MEM_list.append(COMIN_OCEAN_RESTART_MEM)%}
-        {% do COMIN_ICE_ANALYSIS_MEM_list.append(COMIN_ICE_ANALYSIS_MEM)%}
-        {% do COMIN_ICE_LETKF_MEM_list.append(COMIN_ICE_LETKF_MEM)%}
-        {% do COMIN_ICE_HISTORY_MEM_list.append(COMIN_ICE_HISTORY_MEM)%}
-        {% do COMIN_ICE_RESTART_MEM_list.append(COMIN_ICE_RESTART_MEM)%}
-        {% do COMIN_MED_RESTART_MEM_list.append(COMIN_MED_RESTART_MEM)%}
-
-    {% endfor %}
-
-    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
-                          '${RUN}':RUN,
-                          '${YMD}':cycle_YMD,
-                          '${HH}':cycle_HH,
-                          '${MEMDIR}': "ensstat"}) %}
-    {% set COMIN_CONF = COM_CONF_TMPL | replace_tmpl(tmpl_dict) %}
-    # Archive member data
     {% filter indent(width=4) %}
 {% include "enkf_grp.yaml.j2" %}
     {% endfilter %}
 
-    # Determine if restarts should be saved
-    {% set save_warm_start_forecast, save_warm_start_cycled = ( False, False ) %}
-
     # Save the increments and restarts every ARCH_WARMICFREQ days
     # The ensemble increments (group a) should be saved on the ARCH_CYC
     # The forecasts are only run for enkfgdas RUNs, so skip for enkfgfs
-    {% if (current_cycle - SDATE).days % ARCH_WARMICFREQ == 0 and is_gdas %}
-        {% if ARCH_CYC == cycle_HH | int %}
-            {% filter indent(width=4) %}
+    # Note: All archive timing variables (archive_*, save_warm_start_*) are provided by Python via ArchiveTarVars._get_cycle_vars()
+    {% if archive_increments and archive_at_cyc and is_gdas %}
```

Comment:
Is this supposed to be `archive_ics_at_cyc`? If not, what's the difference between `archive_at_cyc` and `archive_ics_at_cyc`?